shells/pipes-shell-2/pipe.js

```diff
@@ -26,32 +29,36 @@ export const initPipe = async (client, paths, storage, composerFactory) => {
   const context = await requireContext();
   // marshal pipes-arc (and stores)
   await marshalPipesArc(storage, context);
-  // construct ShellApi
-  const api = {
-    addPipeEntity(entity) {
-      return addPipeEntity(entity);
-    },
-    async marshalProcessArc(msg, tid, bus) {
-      const composer = composerFactory(msg.modality);
-      const arc = await marshalArc(tid, composer, context, storage, bus);
-      await ingestAndObserve(msg, tid, bus, arc);
-      //log('marshalProcessArc:', arc);
-      return arc;
-    },
-    async marshalSpawnArc(msg, tid, bus) {
-      const composer = composerFactory(msg.modality);
-      const arc = await marshalArc(tid, composer, context, storage, bus);
-      installPlanner(tid, bus, arc, suggestions => deliverSuggestions(tid, bus, suggestions));
-      // create a handle to use as an output slot, forward handle changes to the bus
-      observeOutput(tid, bus, arc);
-      //log('marshalSpawnArc:', arc);
-      return arc;
-    }
-  };
+  // marshal planner
+  initPlanner(context);
```

Comment:
in the PR description, you mentioned that "short-circuiting Planner entirely reduced autofill time to 120ms
" - but this PR only disables speculative execution, or am i reading it wrong?